packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -955,6 +926,16 @@ const VideoPlayerBase = class extends React.Component {
 		return true;
 	}
 
+	handleLoadStart = () => {
+		this.firstPlayReadFlag = true;
+		this.setState({
+			announce: AnnounceState.READY,
```

Comment:
Since `loadstart` event does not fire for the "preload" source, title won't be read.

Reply:
Based on our conversation on disabling playback, should we adjust `Video` to emit the `onLoadStart` event "manually" when switching to a preload source that was already loaded?

Reply:
We could. One question with that is what do you think about handling other loading events? I'm not sure if we have a good reason to only handle `loadstart` event when there are other loading related events like `loadeddata` or `loadedmetadata` and so on. Should we fire those as well?